bundles/org.openhab.binding.tacmi/src/main/java/org/openhab/binding/tacmi/internal/schema/ApiPageParser.java

```diff
@@ -0,0 +1,492 @@
+                                channelType = "Number:Volume";
+                                bd = bd.divide(new BigDecimal(60));
+                                state = new QuantityType<>(bd, SmartHomeUnits.LITRE_PER_MINUTE);
+                            } else {
+                                channelType = "Number";
+                                state = new DecimalType(bd);
+                                logger.debug("Unhandled UoM for channel {} of type {} for '{}': {}", shortName,
+                                        channelType, description, valParts[1]);
+                            }
+                        } else {
+                            channelType = "Number";
+                            state = new DecimalType(bd);
+                        }
+                        if (this.fieldType == FieldType.READ_ONLY || this.address == null) {
+                            ctuid = TACmiBindingConstants.CHANNEL_TYPE_SCHEME_NUMERIC_RO_UID;
+                            type = Type.READ_ONLY_NUMERIC;
+                        } else {
+                            ctuid = null;
+                            type = Type.NUMERIC_FORM;
+                        }
+                    } catch (NumberFormatException nfe) {
+                        // not a number...
+                        channelType = "String";
+                        if (this.fieldType == FieldType.READ_ONLY || this.address == null) {
+                            ctuid = TACmiBindingConstants.CHANNEL_TYPE_SCHEME_STATE_RO_UID;
+                            type = Type.READ_ONLY_STATE;
+                        } else {
+                            ctuid = null;
+                            type = Type.STATE_FORM;
+                        }
+                        state = new StringType(vs);
+                    }
+                }
+                break;
+            case UNKNOWN:
+            case IGNORE:
+                return;
+            default:
+                // should't happen but we have to add default for the compiler...
+                return;
+        }
+        ApiPageEntry e = this.entries.get(shortName);
+        if (e == null || e.type != type || !channelType.equals(e.channel.getAcceptedItemType())) {
+            @Nullable
+            Channel channel = this.taCmiSchemaHandler.getThing().getChannel(shortName);
+            @Nullable
+            ChangerX2Entry cx2e = null;
+            if (this.fieldType == FieldType.FORM_VALUE) {
+                try {
+                    URI uri = this.taCmiSchemaHandler.buildUri("INCLUDE/changerx2.cgi?sadrx2=" + address);
+                    final ChangerX2Parser pp = this.taCmiSchemaHandler.parsePage(uri, new ChangerX2Parser(shortName));
+                    cx2e = pp.getParsedEntry();
+                } catch (final ParseException | InterruptedException | TimeoutException | ExecutionException ex) {
+                    logger.warn("Error loading API Scheme: {} ", ex.getMessage(), ex);
```

Comment:
Can you remove logging the stack trace when the network fails?

Reply:
I've separated the exception to network related problems which are logged without stack trace and parsing exceptions with stack trace. Is this okay? (applied / aligned on two occurrences)